tests/framework/util.go

```diff
@@ -148,3 +137,72 @@ func RemoveDataVolume(dvName string, namespace string) {
 	err = virtCli.CdiClient().CdiV1alpha1().DataVolumes(namespace).Delete(dvName, nil)
 	Expect(err).ToNot(HaveOccurred())
 }
+
+func CreateVMorVMIfromManifest(manifest, nameSpace string) string {
```

Comment:
See tests/framework/types.go 
func (vm VirtualMachine) Create() (string, string, error)  does what you need